README.md

```diff
@@ -24,105 +24,84 @@ It includes the following libraries:
+Set the compilers, if needed, to match those being used for compiling the TPL listed above: `FC`, `CC`, and `CXX` environment variables can be used to point to the desired Fortran, C, and C++ compilers.
 
-Then, run CMake from inside your build directory and pass it the location of the NCEPLIBS-external install directory (if using).
+Execute `cmake` from inside your build directory.
 
-`cmake .. -DEXTERNAL_LIBS_DIR=<path-to-external-libs-install>`
+`cmake -DCMAKE_INSTALL_PREFIX=<nceplibs-prefix> ../NCEPLIBS`
 
-For libraries that are installed by NCEPLIBS-external, no additional settings are required for building the NCEPLIBS other than the `-DEXTERNAL_LIBS_DIR` flag (but see the note on static builds below). For libraries that were installed separately from NCEPLIBS-external, it may be necessary to set the appropriate environment variables so that CMake is able to find them. Please refer to the [NCEPLIBS-external documentation](https://github.com/NOAA-EMC/NCEPLIBS-external) for details.
+If the TPL are not located in a path recognized by `cmake` e.g. `/usr/local`, it may be necessary to provide the appropriate environment variables e.g. `<package_ROOT>` or `CMAKE_PREFIX_PATH` so that `cmake` is able to locate these dependencies.
 
-The default install location is `<build_dir>/install`, but it can be modified by adding:
+The installation prefix for NCEPLIBS is provided by the `cmake` command-line argument `-DCMAKE_INSTALL_PREFIX=<nceplibs-prefix>`
 
-`-DCMAKE_INSTALL_PREFIX=<path>` when running CMake
+To build and install:
 
-When static linking is the default option on a given system (e.g. Cray), add
 ```
--DSTATIC_IS_DEFAULT=ON
+make -j<x>
 ```
-to the `cmake` call.
 
-Then, finally run:
+This will build and install all the NCEPLIBS and place them into the installation prefix `<nceplibs-prefix>`.
 
+Additional `cmake` command-line options while configuring NCEPLIBS:
+`-DFLAT=ON|OFF` - determines the installation tree heirarchy
+`-DDEPLOY=ON|OFF` - prepares and deploys Lua modulefiles for each of the NCEPLIBS libraries.  To deploy modulefiles:
 ```
-make -j<x>
-make install
+make deploy
 ```
+[LMod](https://lmod.readthedocs.io/en/latest/) Environment module system is required to load/unload these modules.
 
-This will make and install all the NCEPLIBS and place them into the install directory.
-
-**Setting your environment to build the UFS Weather Model**
-
-NCEPLIBS produces a shell script that can be sourced to setup your environment to build the UFS Weather Model. Located in `<install>/bin` are `setenv_nceplibs.sh` and `setenv_nceplibs.csh`. Sourcing them will set environment variables that tell the model where to find various libraries such as NCEPLIBS and NetCDF:
-
-`. <install>/bin/setenv_nceplibs.sh`
-
-or, if running C shell,
-
-`source <install>/bin/setenv_nceplibs.csh`
-
-### Troubleshooting
+### Usage
```

Comment:
We need to make sure that any required information specific to the UFS MRW App that is removed here is provided elsewhere, for example as part of the UFS MRW App User's Guide (that's where it belongs I think)

Reply:
Yes.